roboprop_client/demo.sdf.template

```diff
@@ -0,0 +1,106 @@
+<sdf version='1.10'>
+    <world name='shapes'>
+        <plugin filename="gz-sim-physics-system" name="gz::sim::systems::Physics">
+        </plugin>
+        <plugin filename="gz-sim-user-commands-system" name="gz::sim::systems::UserCommands">
+        </plugin>
+        <plugin filename="gz-sim-scene-broadcaster-system" name="gz::sim::systems::SceneBroadcaster">
+        </plugin>
+        <plugin filename="gz-sim-sensors-system" name="gz::sim::systems::Sensors">
+            <render_engine>ogre2</render_engine>
+        </plugin>
+
+        <scene>
+            <ambient>1 1 1 1</ambient>
+            <background>0.800000012 0.800000012 0.800000012 1</background>
+            <shadows>true</shadows>
+        </scene>
+        <plugin name='BoxEnvironment::Plugin'
+            filename='/home/azazdeaz/repos/temp/gz-plugin/examples/build/plugins/libBoxEnvironment.so' />
```

Comment:
What is the demo template for? Is it normal to have local paths referring to @azazdeaz 's machine in it?

Reply:
Sorry and thank you for spotting. The template is used to create a `demo.sdf` of a model that can be loaded in gazebo. It is something left over from Andras' original CLI tool (and refers to the `def add_demo_world(model_path: Path):` function). So not necessary, we could remove the demo logic as a whole, or have the absolute path be dynamic to the person using it. 

Although if Roboprop is something ran on a server rather than a local machine, then perhaps it just needs to be somethingthe user needs to replace themself

Reply:
Sorry for this! I forgot this debug code in the template. Thank you for catching it @fabid!!